### DISCUSSION TITLE
这个响应总太不稳定了、我充plus会员会不会好些了呢、代码里调的接口能识别我是会员吗

### DISCUSSION THREAD

**Discussion by yuzyf** [acheong08/ChatGPT#801] [Q&A] [answered]
求解惑

**Comment by acheong08** [comment, marked as answer]
> 代码里调的接口能识别我是会员吗

会

> 这个响应总太不稳定了

ChatGPT 本身就不太稳定。

Also: I just fixed a server side bug that caused excessive error handling which lead to issues
